assets/scss/_nav.scss

```diff
@@ -24,10 +24,6 @@
     opacity: inherit;
 }
 
-.navbar-announcement-onscroll {
-    position: absolute !important;
-}
-
```

Comment:
This was introduced by #377, which was improved by #413. `.navbar-announcement-onscroll` no longer seems to be used.